src/log/Log.java

```diff
@@ -261,6 +315,28 @@ private static String format(String message, Urgency urgency) {
     }
 
     private static void out(String message) {
-        System.out.println(message);
+        printStream.println(message);
     }
+
+    private static class LogPrintStream extends PrintStream {
+        private final Urgency urgency;
+
+        public LogPrintStream(OutputStream out, Urgency urgency) {
+            super(out);
+            this.urgency = urgency;
+        }
+
+        @Override
+        public void println(Object x) {
+            switch (urgency) {
```

Comment:
I am missing the `default` case for this switch statement.

Reply:
Nice catch. I guess an exception should be thrown like ``colorize``. However, I think it can be dangerous to allow logging to throw exceptions so maybe logging exceptions should just use "Error" as default and ``colorize`` should not use any color?